docs/interceptors.md

```diff
@@ -41,18 +41,18 @@ OkHttpClient client = new OkHttpClient.Builder()
     .build();
 
 Request request = new Request.Builder()
-    .url("http://www.publicobject.com/helloworld.txt")
+    .url("https://www.publicobject.com/helloworld.txt")
     .header("User-Agent", "OkHttp Example")
     .build();
 
 Response response = client.newCall(request).execute();
 response.body().close();
 ```
 
-The URL `http://www.publicobject.com/helloworld.txt` redirects to `https://publicobject.com/helloworld.txt`, and OkHttp follows this redirect automatically. Our application interceptor is called **once** and the response returned from `chain.proceed()` has the redirected response:
+The URL `https://www.publicobject.com/helloworld.txt` redirects to `https://publicobject.com/helloworld.txt`, and OkHttp follows this redirect automatically. Our application interceptor is called **once** and the response returned from `chain.proceed()` has the redirected response:
```

Comment:
Again this changes the meaning of the documentation.  There are two redirects here http->https and www.publicobject.com -> publicobject.com.  Both are intentional.